README.md

```diff
@@ -1,19 +1,102 @@
-Exchange EWS Provider is an **add-on for
-[Thunderbird](https://mozilla.org/thunderbird) and
-[Lightning](https://mozilla.org/calendar)** that provides
-**[Microsoft Exchange](http://microsoft.com/exchange) calendar
-support**.
+## Introduction
+
+Exchange Calendar is an add-on for [Thunderbird](https://mozilla.org/thunderbird)
+and [Seamonkey](https://www.seamonkey-project.org/).
+
+It extends the [Lightning](https://mozilla.org/calendar) add-on to provide
+[Microsoft Exchange](http://microsoft.com/exchange) support.
+
+Exchange Calendar extension can:
+  * Synchronize calendars, tasks, and contacts with a Microsoft Exchange server.
+  * Create, edit and update events and tasks.
+  * Display contacts from your address books and the global address list
+    (they are only readable, they are usable with address autocompletion)
+  * Manage the Exchange « out of office » feature
+
+This extension requires:
+  * Lightning extension corresponding to the Thunderbird release
+  * The Exchange server has to provide an Exchange Web Service
+
+This extension was developed by its original author (Michel Verbraak) for
+Exchange 2007, 2010 and 2013.
+
+The ExchangeCalendar community is now maintaining this extension. Support is
+best effort and cannot be guaranteed. Contributions are welcome.
 
 ## Contributing
 
 This add-on is open source and based on the work of many
 [contributors](https://github.com/ExchangeCalendar/exchangecalendar/graphs/contributors).
 
-### Building
+This project is a community driven effort to keep maintained and uptodate
+the "Exchange EWS Provider" extension created by Michel Verbraak.
+
+Currently, the community is really small and have too few developers to
```

Comment:
Any help to move this project forward is always appreciated.

I would move the warranty sentence into a separate heading "Warrany" before "License" and state:

> This software is provided on a best-effort basis without any warranty. Please see the [license](LICENSE) file for details.